ONSdigital/dp-frontend-release-calendar
# Calendar links to external service to download ICS calendar entries

### What

<img width="254" alt="Screenshot 2022-04-07 at 12 03 21" src="https://user-images.githubusercontent.com/912770/162185059-7910b0e5-0c8d-45b5-b7eb-13fc051cfdc2.png">

Now links to external service generating ICS files to download: https://www.ons.gov.uk/calendar/releasecalendar

### How to review

Code check, or:

- Run dp-design-system in a separate terminal with `make debug`
- Run the frontend with `make debug`
- Visit http://localhost:27700/calendarsample
- Click on the subscription link `Add to your calendar (ICS)`  just above the footer and you should be able to download an ICS file

### Who can review

Anyone
